src/frontend/packages/lib_components/src/data/sideEffects/fetchJitsiInfo/index.ts

```diff
@@ -0,0 +1,24 @@
+import { API_ENDPOINT } from 'settings';
```

Comment:
Why is it declared in lib-coponent ? I would rather leave it to lti-site for now and move it to lib-video once this lib is created

Reply:
It was moved when the upload manager was moved. And we have to fetch the API from both lti-site and standalone-site so I don't understand why it should not be in lib ?

Reply:
I'm not sure to understand the point with upload manager, when upload manager has been migrated to lib-components, api calls required and dedicated to upload manager were also moved to lib components but only them.
Since this call is used for jitsi aka during a live, I would rather migrate it to lib-video in the dedicated PR creating this lib

Reply:
ok